test/do-not-start-memo.test.js

```diff
@@ -42,7 +42,7 @@ test('nested', async (t) => {
   const p = tspl(t, { plan: 5 })
   const a = memo()
 
-  p.deepStrictEqual(a(), undefined)
+  p.throws(() => a(), { message: 'asyncforge store is not initialized for memo1' })
```

Comment:
please add a test for named memo

Reply:
[9091370](https://github.com/mcollina/asyncforge/pull/3/commits/90913709e5563c0ac46fb7232b9cf5f4a6bbc7f6)